pkg/command/command.go

```diff
@@ -190,8 +199,12 @@ func (c *Command) run(printOutput bool) (res *Status, err error) {
 
 			var stdOutWriter, stdErrWriter io.Writer
 			if printOutput {
-				stdOutWriter = io.MultiWriter(os.Stdout, stdOutBuffer)
-				stdErrWriter = io.MultiWriter(os.Stderr, stdErrBuffer)
+				stdOutWriter = io.MultiWriter(append(
```

Comment:
Not blocking, but just wondering if you think we would ever want to support supplying separate writers for `Stdout` and `Stderr`? I could be mistaken here but it looks like if you want one to be written to a passed in writer, than you must allow for both.

Reply:
Yes this gives us more flexibility. I changed the API to allow writers for stdout and stderr separately. Or we just use `AddWriter` for both.

Updated the test cases as well :+1: